### DISCUSSION TITLE
[GitHub.Dev] Color scheme not applied + cannot commit

### DISCUSSION THREAD

**Discussion by avi12** [community/community#54748] [Codespaces]
### Select Topic Area

Bug

### Body

When I click on
![Create Pull Request](https://user-images.githubusercontent.com/6422804/236703177-c5db992c-d259-4bca-98fc-f7efcd27778c.png)
the following panel opens:
![Create Pull Request panel](https://user-images.githubusercontent.com/6422804/236703361-a3c9feb9-b5ca-4b79-ab12-66c469f84207.png)

In addition, when I click <kbd>Create</kbd>:
![Confirmation modal](https://user-images.githubusercontent.com/6422804/236703270-dea02b97-d43b-47c4-8a5c-10bb32a8cfc0.png)
and then I click <kbd>Commit Changes</kbd>:
![this._repository.add is not a function](https://user-images.githubusercontent.com/6422804/236703303-42492781-2c71-419d-99ff-d52022bf5157.png)

**Comment by hunterDroneUp** [comment]
Having this issue as well. 
![image](https://github.com/community/community/assets/118399919/9a70d07c-67cc-4aae-8ed3-38f5e084a07e)
![image](https://github.com/community/community/assets/118399919/fd390de3-7c46-4eb4-aee9-10778c343601)

Solution: 

You must commit your changes under the Source Control section, and then you can create the Pull Request

**Comment by riradog123** [comment]
> ### Select Topic Area
> 
> 
> 
> Bug
> 
> 
> 
> ### Body
> 
> 
> 
> When I click on
> 
> ![Create Pull Request](https://user-images.githubusercontent.com/6422804/236703177-c5db992c-d259-4bca-98fc-f7efcd27778c.png)
> 
> the following panel opens:
> 
> ![Create Pull Request panel](https://user-images.githubusercontent.com/6422804/236703361-a3c9feb9-b5ca-4b79-ab12-66c469f84207.png)
> 
> 
> 
> In addition, when I click <kbd>Create</kbd>:
> 
> ![Confirmation modal](https://user-images.githubusercontent.com/6422804/236703270-dea02b97-d43b-47c4-8a5c-10bb32a8cfc0.png)
> 
> and then I click <kbd>Commit Changes</kbd>:
> 
> ![this._repository.add is not a function](https://user-images.githubusercontent.com/6422804/236703303-42492781-2c71-419d-99ff-d52022bf5157.png)
> 
>
